doc/controls/ResponsiveView.md

```diff
@@ -0,0 +1,131 @@
+---
+uid: Toolkit.Controls.ResponsiveView
+---
+# ResponsiveView
+
+## Summary
+
+The `ResponsiveView` provides the ability to display different content based on screen size, making it easy to adapt to various devices.
+
+### C#
+```csharp
+public partial class ResponsiveView : ContentControl
+```
+
+### XAML
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:ResponsiveView>
+     <!-- Contents -->
+</utu:ResponsiveView>
+```
```

Comment:
this section is completely useless...
remove

Reply:
Done in my next commit